.github/workflows/pr-checks.yml

```diff
@@ -86,4 +86,29 @@ jobs:
         with:
           name: testDebugUnitTest
           path: ./**/build/reports/tests/testDebugUnitTest
-      
\ No newline at end of file
+  ui_test:
```

Comment:
I would say no. It would increase our build time a lot. I put it here so we can discuss where to move it. (Maybe we can create a con job that runs this actions every night)